pkg/netperf/netperf.go

```diff
@@ -49,7 +50,7 @@ func Run(c *kubernetes.Clientset, rc rest.Config, nc config.Config, client apiv1
 			fmt.Sprintf("%d", nc.Duration),
 			"-t", nc.Profile,
 			"--",
-			"-k", fmt.Sprintf("%s", omniOptions),
+			"-k", fmt.Sprint(omniOptions),
 			"-m", fmt.Sprintf("%d", nc.MessageSize),
```

Comment:
You can improve this statement too
```suggestion
			"-m", fmt.Sprint(nc.MessageSize),
```

Reply:
Updated